app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/Invoices.php

```diff
@@ -14,6 +17,27 @@
 class Invoices extends \Magento\Framework\View\Element\Text\ListText implements
     \Magento\Backend\Block\Widget\Tab\TabInterface
 {
+    /**
+     * @var AuthorizationInterface
+     */
+    private $authorization;
+
+    /**
+     * Invoices constructor.
+     *
+     * @param Context $context
+     * @param AuthorizationInterface $authorization
+     * @param array $data
+     */
+    public function __construct(
+        Context $context,
+        AuthorizationInterface $authorization,
```

Comment:
This class is marked with @api class, so we can't introduce any backward incompatible changes.

Could you make this argument the last one, make it optional with fallback to retrieving this object from the object manager?

Reply:
Done